app/controllers/projects/contributions_controller.rb

```diff
@@ -57,6 +59,7 @@ def create
     @contribution.value = permitted_params[:contribution][:value]
     @contribution.referal_link = permitted_params[:contribution][:referal_link]
     @contribution.reward_id = (params[:contribution][:reward_id].to_i == 0 ? nil : params[:contribution][:reward_id])
+    session[:fallback_redirect] = fallback_create_project_contributions_path(params) unless current_user.present?
```

Comment:
Unnecessary, if we have a GET route for create, the default redirect should work.